public/app/features/alerting/unified/components/rule-editor/alert-rule-form/AlertRuleForm.tsx

```diff
@@ -147,16 +147,16 @@ export const AlertRuleForm = ({ existing, prefill }: Props) => {
     if (!existing) {
       // when creating a new rule, we save the manual routing setting , and editorSettings.simplifiedQueryEditor to the local storage
       storeInLocalStorageValues(values);
-      await addRuleToRuleGroup.execute(ruleGroupIdentifier, ruleDefinition, values.evaluateEvery);
+      await addRuleToRuleGroup.execute(ruleGroupIdentifier, ruleDefinition, evaluateEvery);
     } else {
       const ruleIdentifier = fromRulerRuleAndRuleGroupIdentifier(ruleGroupIdentifier, existing.rule);
       const targetRuleGroupIdentifier = getRuleGroupLocationFromFormValues(values);
-
       await updateRuleInRuleGroup.execute(
         ruleGroupIdentifier,
         ruleIdentifier,
         ruleDefinition,
-        targetRuleGroupIdentifier
+        targetRuleGroupIdentifier,
+        evaluateEvery
```

Comment:
This is the actual fix - passing interval in, and using it from the state rather than the values. For some (older) reason, we aren't only tracking the evaluateEvery in the form state, but also in a `useState` 🤷 The useState one is the value that's actually updated as we select items in the form